_topic_map.yml

```diff
@@ -2750,6 +2750,8 @@ Topics:
       File: virt-schedule-vms
     - Name: Configuring PCI passthrough
       File: virt-configuring-pci-passthrough
+    - Name: Planning object maximums
+      File: virt-planning-object-maximums
```

Comment:
I'll synch up the title and assembly.

About placement - that's a great point about the customer needing this info early on. 

Couple of points to note - in the 4.9 release this content is going to be substantive and robust. 

For the 4.8 release, we want to get out the basic info out about our testing results. The "Preparing your cluster for virt" assembly provides recommendations that are neatly categorized into storage, network etc. That content is mature and comprehensive.

The content for the current Jira story is limited and it shares what was tested and the results of the testing. Given the smaller scope of the content for this story, adding it it in a meaningful way into the "Preparing your cluster" section will be challenging. 

And the main OpenShift docs separate all of the testing info for object maximums into a standalone assembly.

I anticipate that for the 4.9 release, this entire assembly will have multiple modules with content relevant to tested scenarios for different cloud platforms and several tables with data on what was tested and the results. In light of these facts, adding this content to the Advanced VM Management and also because the information is very limited for this release. Moving this content to a more meaningful presentation (when we have more testing results) for the 4.9 release is definitely a goal in place.

Reply:
Synched up the title and the assembly title.
Thanks for the excellent feedback about the proper home for this content. I moved this assembly to the location where we have install-related content. I am looking forward to the 4.9 release where the content for this story will be beefed up significantly with cloud platform test rests and more.